server/apps/distributed-app/helm-chart/james/configs/blob.properties

```diff
@@ -0,0 +1,81 @@
+# ============================================= BlobStore Implementation ==================================
```

Comment:
And it's because of the reason that people download the whole thing and start testing that I think we should add the mailetcontainer.xml file too here :)